### ISSUE TITLE
Order by percentiles isn't working

### ISSUE THREAD

**Issue by maxcleme** [grafana/opensearch-datasource#341]
https://github.com/grafana/opensearch-datasource/assets/7669401/24046232-1925-498c-bb24-67f55693f290

**What happened**:

I'm in the middle of migrating panels from using Elasticsearch Datasource to OpenSearch Datasource. However I can't properly Order By percentile Metrics in my Group By aggregation. When doing such, I get `Invalid aggregation order path [1]. When ordering on a multi-value metrics aggregation a metric name must be specified.` error. It works fine using the same setup with ES datasource.

It's also worth mentioning that the suggestion on the Order by is a bit different from Elasticsearch Datasource:

**Opensearch**

![Screenshot 2024-02-29 at 16 33 58](https://github.com/grafana/opensearch-datasource/assets/7669401/d1ca39b0-e3ac-4a13-82fb-3d96e8c6de44)

**Elasticsearch**

![Screenshot 2024-02-29 at 16 34 21](https://github.com/grafana/opensearch-datasource/assets/7669401/15df4076-3444-4b30-a842-1e0edce28592)

**What you expected to happen**:

I would have expect it to work, as it works in ES datasource.

**How to reproduce it (as minimally and precisely as possible)**:

I guess video submitted shows it all.

**Anything else we need to know?**:

**Environment**:
- Grafana version: `Grafana v10.2.3 (1e84fede54)`
- OpenSearch version: `2.11`
- Plugin version: `2.14.4`

**Comment by maxcleme** [comment]
Not sure why it got tagged `AWS Datasource` 😅

**Comment by iwysiu** [comment]
Hi @maxcleme ! Thanks for reporting this bug. I confirmed that it works in the ElasticSearch plugin and not in the OpenSearch plugin. I'll move it to our backlog to be looked at.
Investigation notes: For reproducing it, I ran `make devenv sources=elastic` to run the ElasticSearch datasource and used the `gdev-elasticsearch-v2-logs` datasource

**Comment by maxcleme** [comment]
Yes, the issue is still relevant and has been acknowledge above.

**Comment by idastambuk** [comment]
Hi @maxcleme and @almog-orca thank you for your patience, we just released v2.26.0 where this was fixed
